lib/__tests__/Env.ts

```diff
@@ -2,20 +2,14 @@ import { readEnvironment } from '../Helper/Env';
 
 describe('read-env', () => {
   test('transform', () => {
-    // @ts-ignore: true not assignable to string/undefined
-    process.env.SENTRY_WIZARD_DEBUG = true;
-    // @ts-ignore: true not assignable to string/undefined
-    process.env.SENTRY_WIZARD_UNINSTALL = false;
-    // @ts-ignore: true not assignable to string/undefined
-    process.env.SENTRY_WIZARD_SKIP_CONNECT = true;
-    // @ts-ignore: true not assignable to string/undefined
-    process.env.SENTRY_WIZARD_QUIET = true;
-    // @ts-ignore: true not assignable to string/undefined
-    process.env.SENTRY_WIZARD_INTEGRATION = ['reactNative', 'electron'];
-    // @ts-ignore: true not assignable to string/undefined
-    process.env.SENTRY_WIZARD_PLATFORM = ['ios', 'android'];
-    // @ts-ignore: true not assignable to string/undefined
+    process.env.SENTRY_WIZARD_DEBUG = 'true';
```

Comment:
TS complained rightfully here. Changing the values to string here is the right thing to do and the test still passes (meaning `read-env` did its job)